homeassistant/components/epic_games_store/config_flow.py

```diff
@@ -0,0 +1,85 @@
+"""Config flow for Epic Games Store integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from epicstore_api import EpicGamesStoreAPI
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.selector import LanguageSelector, LanguageSelectorConfig
+
+from .const import CONF_LANGUAGE, CONF_SUPPORTED_LANGUAGES, DOMAIN
```

Comment:
`CONF_LANGUAGE` is available in `homeassistant.const`.

Reply:
Done in 724ce255805e57ea05a7040f6e19d2b945ff67c7